e2e/app/base-page.ts

```diff
@@ -232,7 +232,34 @@ export default abstract class BasePage {
     const timestamp = new Date().getTime();
     const screenshotFile = `${SCREENSHOT_DIR}/${fileName}_${timestamp}.png`;
     await this.page.screenshot({path: screenshotFile, fullPage: true});
-    console.log('screenshot taken: ' + screenshotFile);
+    console.log('Saved screenshot ' + screenshotFile);
+  }
+
+  async saveToFile(fileName, data, suffix: string = 'html') {
+    const LOG_DIR = 'logs/html';
```

Comment:
In the codebase we typically don't capitalize local variables, even if they are constants.